packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -107,6 +110,25 @@ private void updateSettings(MethodCall methodCall, Result result) {
     result.success(null);
   }
 
+  private void evaluateJavaScript(MethodCall methodCall, final Result result) {
+    if (!webView.getSettings().getJavaScriptEnabled()) {
+      throw new UnsupportedOperationException("JavaScript Mode disabled");
+    } else {
+      String jsString = (String) methodCall.arguments;
+      if (jsString == null) {
+        throw new UnsupportedOperationException("JavaScript String cannot be null");
+      }
+      webView.evaluateJavascript(
+          jsString,
+          new android.webkit.ValueCallback<String>() {
+            @Override
+            public void onReceiveValue(String value) {
+              result.success("EvaluateJavaScript result: " + value);
```

Comment:
Wouldn't it be easier to parse the result without the `"EvaluateJavascript result: " prefix?

We should also demonstrate usage of the result value in the example app.